src/components/Button/index.js

```diff
@@ -6,7 +6,7 @@ import style from './index.module.scss';
 /*
     To use this component, pass the following props:
     
-    > buttonLabel : Pass a string value for the button label.
+    > buttonLabel : Pass a string value or a function for the button label .
```

Comment:
do we really need to pass a function for the label? can we not just pass a string?